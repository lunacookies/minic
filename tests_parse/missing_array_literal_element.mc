func main {
	a := [1,, 2]
}
