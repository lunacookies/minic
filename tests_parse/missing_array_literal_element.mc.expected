func main {
	a := [
		1,
		<missing>,
		2,
	]
}
tests_parse:21..22: error: missing array element
